### ISSUE TITLE
broken email validation in 3.19.0

### ISSUE THREAD

**Issue by JoueBien** [mikeerickson/validatorjs#381]
In 3.19.0 emails with an extra extension on the end cause the validation to incorrectly fail.
Example of what still Works:
```
const rules={ email: 'required|email' }

test@test.com
```

Example of what now incorrectly Fails:
```
const rules={ email: 'required|email' }

test@test.com.au
test@test.org.uk
```

Those last to emails will correctly pass validation in 3.18.1

**Comment by mikeerickson** [comment]
I am working on a quick update 3.19.1 and I will include the fix for this in that update.  Should be ready by end of weekend.

**Comment by echart** [comment]
+1

**Comment by MathewsCoelho** [comment]
+1

**Comment by mikeerickson** [comment]
@JoueBien I just published a new release `3.19.1` to address this issue.
